src/dataSources/JSON.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+var Null = require('./Null');
+
+/**
+ * @constructor
+ */
+var JSON = Null.extend('JSON',  {
```

Comment:
I don't like calling it JSON. Calling the behavior and dataModel JSON was inaccurate at best and not such a good choice. How about DataSource or RawDataSource or something more descriptive?